### PR TITLE
Feature/2 refactoring

### LINKED ISSUE #2: Refactor initial commit
PR #5 

- [x] introduce `components` directory
- [x] create `Header` component as functional component (with arrow function)
- [x] rename file `importData` to `DataList` and move to `components`, also rename to `Body`
- [x] create `util` directory
  - introduce `parseCsv` function 
      - input: `csvData`: String
      - output: `dataList`
      - skip header line
      - create tests
- [x] make `data` directory for `textfile`
     - rename `textfile` to `data.csv` 
         - remove import in `Body`
     - move `adverity_logo.svg` into public folder 
        - update import in `App.js`
- [x] move `TypefillSearchBar.js` to `components`

### PR DESCRIPTION
**Pull request by jasrial**
Closes #2

### CODE REVIEW

File: src/components/Body.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import ParseCsv from "../utils/ParseCsv";
+
+class Body extends Component {
+  state = {
+    objectlist: []
+  };
+
+  componentDidMount() {
+    axios.get("http://typeaheadaggregates.free.beeceptor.com").then(data => {
+      const simplelist = data.data;
```
Comment (vlabs) [inline src/components/Body.js]: find a better name please or directly assign in below

File: src/components/Body.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import React, { Component } from "react";
+import axios from "axios";
+import Select from "react-select";
+import ParseCsv from "../utils/ParseCsv";
+
+class Body extends Component {
+  state = {
+    objectlist: []
```
Comment (vlabs) [inline src/components/Body.js]: rename to `dataList`

File: src/utils/ParseCsv.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+
+function ParseCsv(csvData){
```
Comment (vlabs) [inline src/utils/ParseCsv.js]: camelCase with first letter lowercase

File: src/utils/ParseCsv.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    var csvData_splitted = csvData.split('\n');
```
Comment (vlabs) [inline src/utils/ParseCsv.js]: no snakecase

File: src/utils/ParseCsv.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+    csvData_splitted.splice(0, 1);
+
+    return csvData_splitted.map(item =>{
+        const list_element = item.split(',');
+        const object = {}
+        object.campaign = list_element[0];
+        object.channel = list_element[1];
+        object.clicks = Number(list_element[2]);
+        object.impressions = Number(list_element[3]);
+        return object;
```
Comment (vlabs) [inline src/utils/ParseCsv.js]: `return { campaign: list_element[0], ..., impressions = Number(list_element[3]) }`

File: src/components/Body.js (hunk 1 above)
Comment (jasrial) [reply src/components/Body.js]: 4bb8aec8eadeb3a3c247c574f735f9aba11c66c7

File: src/utils/ParseCsv.js (hunk 3 above)
Comment (jasrial) [reply src/utils/ParseCsv.js]: db2e6f175e428bcd28e483b8a513ad6c99126d4b

Comment (jasrial) [reply src/utils/ParseCsv.js]: commit db2e6f175e428bcd28e483b8a513ad6c99126d4b

File: src/utils/parseCsv.js [hunk 6]
```diff
@@ -0,0 +1,11 @@
+function parseCsv(csvData) {
+  var csvDataSplitted = csvData.split("\n");
+  csvDataSplitted.splice(0, 1);
```
Comment (vlabs) [inline src/utils/parseCsv.js]: don't use mutating functions/methods

File: src/components/Body.js (hunk 2 above)
Comment (jasrial) [reply src/components/Body.js]: 48de2088e52c2db3fe450d833c0b652f33bb14a6

File: src/utils/ParseCsv.js (hunk 3 above)
Comment (jasrial) [reply src/utils/ParseCsv.js]: db2e6f175e428bcd28e483b8a513ad6c99126d4b

File: src/utils/ParseCsv.js (hunk 4 above)
Comment (jasrial) [reply src/utils/ParseCsv.js]: 08586a0f2cc607511980138358c9c912025b3d3b

File: src/utils/ParseCsv.js (hunk 5 above)
Comment (jasrial) [reply src/utils/ParseCsv.js]: 0984bf1e2c48ea4afaa2faef83b0d07496a7bd90

File: src/utils/parseCsv.js (hunk 6 above)
Comment (jasrial) [reply src/utils/parseCsv.js]: d134a9b62de2dffc2002e2a3759e4f449462c1fa

**Pull request merged**